yuniql-cli/BaseConfigOption.cs

```diff
@@ -0,0 +1,17 @@
+﻿using CommandLine;
+
+namespace Yuniql.CLI
+{
+    public class BaseConfigOption : BasePlatformOption
+    {
+        //yuniql <command> -o json
+        [Option('o', "data-type", Required = false, HelpText = "The choosen Output format")]
```

Comment:
Lets "output" instead of "data-type", this is more consistent with many CLI tools like aws cli and kubectl cli.

Reply:
totally agree 👍